From: =?utf-8?q?Mohammed_Adn=C3=A8_Trojette?= <adn+deb@diwi.org>
Date: Sun, 4 Jun 2006 10:43:06 +0200
Subject: 01 makefile

Makefiles cleanup
---
 makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/makefile b/makefile
index 5479b48..e97764f 100644
--- a/makefile
+++ b/makefile
@@ -81,7 +81,6 @@ sfx: common
 common7z:common
 	$(MKDIR) bin/Codecs
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree all
-	$(MAKE) -C CPP/7zip/Compress/Rar         all
 
 7z: common7z
 	$(MAKE) -C CPP/7zip/UI/Console           all
@@ -106,7 +105,6 @@ clean:
 	$(MAKE) -C CPP/7zip/UI/FileManager       clean
 	$(MAKE) -C CPP/7zip/UI/GUI               clean
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree clean
-	$(MAKE) -C CPP/7zip/Compress/Rar         clean
 	$(MAKE) -C CPP/7zip/Bundles/LzmaCon      clean2
 	$(MAKE) -C CPP/7zip/Bundles/AloneGCOV    clean
 	$(MAKE) -C CPP/7zip/TEST/TestUI          clean
