From: =?utf-8?q?Mohammed_Adn=C3=A8_Trojette?= <adn+deb@diwi.org>
Date: Sun, 4 Jun 2006 10:43:06 +0200
Subject: Makefiles cleanup

---
 makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/makefile b/makefile
index 745c8ed..a936f03 100644
--- a/makefile
+++ b/makefile
@@ -42,7 +42,6 @@ sfx: common
 common7z:common
 	$(MKDIR) bin/Codecs
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree all
-	$(MAKE) -C CPP/7zip/Compress/Rar         all
 
 lzham:common
 	$(MKDIR) bin/Codecs
@@ -67,7 +66,6 @@ clean_C:
 	$(MAKE) -C CPP/7zip/UI/FileManager       clean
 	$(MAKE) -C CPP/7zip/UI/GUI               clean
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree clean
-	$(MAKE) -C CPP/7zip/Compress/Rar         clean
 	$(MAKE) -C CPP/7zip/Compress/Lzham       clean
 	$(MAKE) -C CPP/7zip/Bundles/LzmaCon      clean2
 	$(MAKE) -C CPP/7zip/Bundles/AloneGCOV    clean
