Index: p7zip-4.47~dfsg.1/makefile
===================================================================
--- p7zip-4.47~dfsg.1.orig/makefile	2007-06-08 10:54:43.000000000 +0200
+++ p7zip-4.47~dfsg.1/makefile	2007-06-08 10:56:35.000000000 +0200
@@ -41,7 +41,6 @@
 	cd CPP/7zip/UI/Console        ; $(MAKE) depend
 	cd CPP/7zip/Bundles/Format7zFree ; $(MAKE) depend
 	cd CPP/7zip/Bundles/Format7z  ; $(MAKE) depend
-	cd CPP/7zip/Compress/Rar      ; $(MAKE) depend
 
 sfx: common
 	mkdir -p  bin
@@ -51,7 +50,6 @@
 	mkdir -p  bin/Codecs
 	cd CPP/7zip/UI/Console           ; $(MAKE) all
 	cd CPP/7zip/Bundles/Format7zFree ; $(MAKE) all
-	cd CPP/7zip/Compress/Rar         ; $(MAKE) all
 
 clean:
 	cd CPP/Common                    ; $(MAKE) clean
@@ -64,7 +62,6 @@
 	cd CPP/7zip/Bundles/Format7zF    ; $(MAKE) clean
 	cd CPP/7zip/Bundles/Format7zFree ; $(MAKE) clean
 	cd CPP/7zip/Bundles/Format7z     ; $(MAKE) clean
-	cd CPP/7zip/Compress/Rar         ; $(MAKE) clean
 	cd CPP/7zip/Compress/LZMA_Alone  ; $(MAKE) clean
 	cd CPP/7zip/Bundles/AloneGCOV    ; $(MAKE) clean
 	rm -fr bin
