Index: p7zip-9.20.1~dfsg.1/makefile
===================================================================
--- p7zip-9.20.1~dfsg.1.orig/makefile	2011-04-01 21:14:01.000000000 +0200
+++ p7zip-9.20.1~dfsg.1/makefile	2011-04-01 21:42:45.000000000 +0200
@@ -68,7 +68,6 @@
 common7z:common
 	$(MKDIR) bin/Codecs
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree all
-	$(MAKE) -C CPP/7zip/Compress/Rar         all
 
 7z: common7z
 	$(MAKE) -C CPP/7zip/UI/Console           all
@@ -93,7 +92,6 @@
 	$(MAKE) -C CPP/7zip/UI/FileManager       clean
 	$(MAKE) -C CPP/7zip/UI/GUI               clean
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree clean
-	$(MAKE) -C CPP/7zip/Compress/Rar         clean
 	$(MAKE) -C CPP/7zip/Compress/LZMA_Alone  clean
 	$(MAKE) -C CPP/7zip/Bundles/AloneGCOV    clean
 	$(MAKE) -C CPP/7zip/TEST/TestUI          clean
Index: p7zip-9.20.1~dfsg.1/makefile.machine
===================================================================
--- p7zip-9.20.1~dfsg.1.orig/makefile.machine	2011-04-01 21:14:01.000000000 +0200
+++ p7zip-9.20.1~dfsg.1/makefile.machine	2011-04-01 21:36:34.000000000 +0200
@@ -4,7 +4,7 @@
 
 OPTFLAGS=-O
 
-ALLFLAGS=${OPTFLAGS} -pipe -s \
+ALLFLAGS=${OPTFLAGS} -pipe \
     	-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE \
 	-DNDEBUG -D_REENTRANT -DENV_UNIX \
 	-D_7ZIP_LARGE_PAGES \
