Index: p7zip-4.58~dfsg.1/makefile
===================================================================
--- p7zip-4.58~dfsg.1.orig/makefile	2008-06-21 15:28:28.000000000 +0200
+++ p7zip-4.58~dfsg.1/makefile	2008-06-21 15:30:42.000000000 +0200
@@ -41,7 +41,6 @@
 	cd CPP/7zip/UI/Client7z       ; $(MAKE) depend
 	cd CPP/7zip/UI/Console        ; $(MAKE) depend
 	cd CPP/7zip/Bundles/Format7zFree ; $(MAKE) depend
-	cd CPP/7zip/Compress/Rar      ; $(MAKE) depend
 
 sfx: common
 	$(MKDIR) bin
@@ -51,7 +50,6 @@
 	$(MKDIR) bin/Codecs
 	cd CPP/7zip/UI/Console           ; $(MAKE) all
 	cd CPP/7zip/Bundles/Format7zFree ; $(MAKE) all
-	cd CPP/7zip/Compress/Rar         ; $(MAKE) all
 
 clean:
 	cd CPP/myWindows                 ; $(MAKE) clean
@@ -61,7 +59,6 @@
 	cd CPP/7zip/UI/Client7z          ; $(MAKE) clean
 	cd CPP/7zip/UI/Console           ; $(MAKE) clean
 	cd CPP/7zip/Bundles/Format7zFree ; $(MAKE) clean
-	cd CPP/7zip/Compress/Rar         ; $(MAKE) clean
 	cd CPP/7zip/Compress/LZMA_Alone  ; $(MAKE) clean
 	cd CPP/7zip/Compress/PPMD_Alone  ; $(MAKE) clean
 	cd CPP/7zip/Bundles/AloneGCOV    ; $(MAKE) clean
