Index: p7zip-4.51/CPP/Windows/System.cpp
===================================================================
--- p7zip-4.51.orig/CPP/Windows/System.cpp	2007-08-04 20:54:26.000000000 +0200
+++ p7zip-4.51/CPP/Windows/System.cpp	2007-08-04 21:10:41.000000000 +0200
@@ -1,7 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#if defined (__NetBSD__) || defined(__OpenBSD__) || defined (__FreeBSD__) || defined (__APPLE__)
+#if defined (__NetBSD__) || defined(__OpenBSD__) || defined (__FreeBSD__) || defined (__FreeBSD_kernel__) || defined (__APPLE__)
 #include <sys/param.h>
 #include <sys/sysctl.h>
 #elif defined(__linux__) || defined(__CYGWIN__) || defined(sun)
@@ -32,7 +32,7 @@
 						  		nbcpu = value;
 			return nbcpu;
 		}
-		#elif defined (__FreeBSD__)
+		#elif defined (__FreeBSD__) || defined (__FreeBSD_kernel__)
 		UInt32 GetNumberOfProcessors() {
 		  		int nbcpu = 1;
 			size_t value;
